scripts/generate_input_continuum.ipynb

```diff
@@ -50,13 +50,13 @@
    "source": [
     "# load arguments\n",
     "data_file = os.path.join(os.path.dirname(os.getcwd()), \"data\", \"all-pseudomonas-gene-normalized.zip\")  # repo file is zipped\n",
-    "map_file = os.path.join(os.path.dirname(os.getcwd()), \"metadata\", \"mapping_oxy.txt\")\n",
+    "map_file = os.path.join(os.path.dirname(os.getcwd()), \"metadata\", \"mapping_PA1673.txt\")\n",
```

Comment:
there are a couple instances in the subsequent files where I would make a small aesthetic suggestion. Where you have multiple paths with the same base directory, set the base first in a new variable. For example:

```python
base_dir = os.path.join(os.path.dirname(os.getcwd()), 'metadata')
map_file = os.path.join(base_dir, 'mapping_PA1673.txt')
```

Reply:
I will address this with the refactor PR as I will be re-organizing files for that